guides/integrate-nextauth.md

```diff
@@ -0,0 +1,427 @@
+```
+
+Once you receive the incoming URL, pass it into the SDK to complete the binding process. You can validate the incoming URL with `isBindPasskeyUrl`. Upon success, a private key will have been created in the device's hardware trust module and the corresponding public key will have been sent to the Beyond Identity Cloud. At this point the user has a passkey enrolled on this device.
+
+Create a `bind.tsx` page under `/next-auth-example/pages`. As long as your `Invoke URL` is configured properly in your [Authenticator Config](/docs/v1/platform-overview/authenticator-config), this is the page that will be redirected to during a bind passkey flow. Copy the following code snippet into that page.
+
+<Tabs groupId="nextjs" queryString>
+<TabItem value="nextauth" label="NextAuth.js">
+
+```javascript
+import { useEffect, useState } from "react";
+import "bootstrap/dist/css/bootstrap.css";
+import { signIn } from "next-auth/react";
+import { Passkey } from "@beyondidentity/bi-sdk-js";
+
+const BIBindPasskey = () => {
+  const [bindPasskeyResult, setBindPasskeyResult] = useState('');
+
+  useEffect(() => {
+    // -- 1
+    const bindPasskey = async () => {
+      const BeyondIdentityEmbeddedSdk = await import("@beyondidentity/bi-sdk-js");
+      let embedded = await BeyondIdentitySdk.EmbeddedSdk.initialize();
+      if (embedded.isBindPasskeyUrl(window.location.href)) {
+        // Only bind passkey if the URL is a "bind" URL
+        let bindPasskeyUrl = window.location.href;
+        // -- 2
+        embedded
+          .bindPasskey(bindPasskeyUrl)
+          .then((result) => {
+            // -- 3
+            setBindPasskeyResult(result);
+            signIn('beyondidentity', {
+              tenant_id: tenantId,
+            });
+          })
+          .catch((error) => {
+            setBindPasskeyResult(error.toString());
+          });
+      }
+    };
+
+    bindPasskey().catch(console.error);
+  }, []);
+
+  return (
+    <div
+      style={{
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'center',
+        height: '100vh',
+      }}
+    >
+      <div className="container">
+         <div className="row">	
+          <div className="d-flex justify-content-center">	
+            <div className="spinner-border" role="status">	
+              <span className="sr-only"></span>	
+            </div>	
+          </div>	
+        </div>
+        <div className="row">
+          {bindPasskeyResult.length > 0 && (
+            <div className="row row-cols-1 row-cols-md-1 mt-3">
+              <div className="col">
+                <code>
+                  {JSON.stringify(bindPasskeyResult, null, 2)}
+                </code>
+              </div>
+            </div>
+          )}
+        </div>
+      </div>
+    </div>
+  );
+};
+
+export default BIBindPasskey;
+```
+
+</TabItem>
+<TabItem value="auth" label="Auth.js">
+
+:::warning WIP
```

Comment:
Can we make a jira ticket to finish documenting Auth.js when it's ready? There's a ticket to implement that in Acme, maybe they could be linked?

Reply:
There is 🙂 I can link it to that ticket